changelogs/unreleased/7849-code-loading.yml

```diff
@@ -0,0 +1,3 @@
+description: Make sure the compiler will not load Python code code present in the `files`, `model` or `templates` directory
+change-type: patch
+destination-branches: [master, iso7]
```

Comment:
This is technically a breaking change because not only do we fix the bug, we also prohibit the `files` and `templates` namespaces.

I think we have to make this iso8 only. @wouterdb wdyt?

Reply:
It would be really nice to have this bug fixed in iso7 as well. It has bitten us already a few times.

Reply:
Then we break that hypothetical module that we are sure of that does not exist.

Reply:
We need this in iso7, it really is a bug.

(and the module build would crash if there is both a files folder and a files submodule )